misk/src/main/kotlin/misk/util/Checks.kt

```diff
@@ -0,0 +1,11 @@
+package misk.util
+
+object Checks {
```

Comment:
Why not just Kotlin check and require ?

Reply:
https://github.com/JetBrains/kotlin/blob/1.2.0/libraries/stdlib/src/kotlin/util/Preconditions.kt#L13

Reply:
Wasn't aware of require. Spiff